cs/v1/CS_v1.0.0.yaml

```diff
@@ -323,11 +352,11 @@ paths:
         - schema:
             type: string
             maxLength: 7
```

Comment:
Max length is no longer 7 please change to 8. I have added a `minLength` of 7 also